libs/dh/shared/feature-graphql-codegen/dist/index.js

```diff
@@ -14,7 +17,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-'use strict';
-Object.defineProperty(exports, '__esModule', { value: true });
-const tslib_1 = require('tslib');
-tslib_1.__exportStar(require('./apollo-data-source'), exports);
```

Comment:
nitpick: the license header can be removed. Strange why it wasn't removed automatically when the file was recreated but anyway.

Reply:
Yeah I noticed that, it seems to be the `tsc` command that doesn't remove them. Not sure why it keeps that comment when it removes it from the other file 🤔 Anyway, I didn't think it was important enough to bother fixing, I'd rather just keep the default output of `tsc`